doc/modules/language-guide/pages/basic-concepts.adoc

```diff
@@ -267,7 +267,7 @@ Another mathematical way to think of the unit value is as a tuple with no elemen
 
 === Natural numbers
 
-The members of this type consist of the usual values - `0`, `1`, `2`, ... - but, as in mathematics, the members of are not bound to a special maximum size.
+The members of this type consist of the usual values - `0`, `1`, `2`, ... - but, as in mathematics, the members `Nat` of are not bound to a special maximum size.
```

Comment:
"`Nat` of" -> "of `Nat`"?

Reply:
Updated in a new PR https://github.com/dfinity/motoko/pull/2990

Thank you for pointing it out